src/es_archiver/lib/docs/__tests__/index_doc_records_stream.ts

```diff
@@ -26,12 +26,12 @@ import { Progress } from '../../progress';
 import { createIndexDocRecordsStream } from '../index_doc_records_stream';
 import { createStubStats, createStubClient, createPersonDocRecords } from './stubs';
 
-const recordsToBulkBody = records => {
+const recordsToBulkBody = (records: any[]) => {
   return records.reduce((acc, record) => {
     const { index, id, source } = record.value;
 
     return [...acc, { index: { _index: index, _id: id } }, source];
-  }, []);
+  }, [] as any[]);
```

Comment:
the `elasticsearch` node lib does not have proper typing for query responses. every records, indices and/or any result from ES are `any`. Was forced to 'follow'